src/main/charts/confluence/values.yaml

```diff
@@ -81,14 +87,22 @@ serviceAccount:
     # -- Set to 'true' if a ClusterRole should be created, or 'false' if it
     # already exists.
     #
-    create: true
+    create: false
```

Comment:
What will happen to existing installations, if they upgrade to the new version and they didn't override the default, it will delete the ClusterRole and create Role?

Reply:
Yes, correct. ClusterRole and ClusterRoleBinding will be deleted, instead Role and RoleBinding created.

Reply:
Updated PR description